server/src/main/java/org/elasticsearch/indices/recovery/RecoveryState.java

```diff
@@ -414,9 +424,16 @@ public synchronized void writeTo(StreamOutput out) throws IOException {
 
     }
 
-    public static class VerifyIndex extends Timer implements ToXContentFragment, Streamable {
+    public static class VerifyIndex extends Timer implements ToXContentFragment, Streamable, Writeable {
```

Comment:
I think we can remove `Streamable` from `VerifyIndex` then remove `readFrom` entirely.